inc/search.class.php

```diff
@@ -3683,9 +3827,11 @@ static function addSelect($itemtype, $ID, $meta = 0, $meta_type = 0) {
     *
     * @param string $itemtype device type
     *
-    * @return string Where string
+    * @return string
+    *
+    * @since 10.0.0 Method is no longer static
    **/
-   static function addDefaultWhere($itemtype) {
+   public function addDefaultWhere($itemtype) {
```

Comment:
This method is used in some plugins (`formcreator` and `fusioninventory` for instance), change should be listed in changelog.